[DOCS] Updating Contribute to OpenVINO articles (#23650)
openvinotoolkit/openvino: docs/articles_en/about-openvino/contributing.rst

Contribute to OpenVINO
========================

.. toctree::
   :maxdepth: 1
   :hidden:

   contributing/code-contribution-guide

OpenVINO™ is always looking for opportunities to improve and your contributions
play a big role in this process. Here are four ways you can make OpenVINO better:

- `Provide feedback <#provide-feedback>`__
- `Contribute code changes <#contribute-code-changes>`__
- `Improve documentation <#improve-documentation>`__
- `Promote and support OpenVINO <#promote-and-support-openvino>`__

> 

Reviewer: - Pick up a Good First Issue?

Author: It is strongly highlighted in "Code Contribution Guide".

:fas:`comments` Provide feedback
################################

.. rubric:: Report bugs / issues
   :name: report-bugs-issues

If you notice unexpected behavior in OpenVINO or its components, you can
`create a new issue <https://github.com/openvinotoolkit/openvino/issues>`__
in the GitHub issue tracker.

.. rubric:: Propose improvements
   :name: propose-improvements

If you want to share your ideas for improving OpenVINO:

- Open a new `GitHub Discussion <https://github.com/openvinotoolkit/openvino/discussions>`__.
- Create a `Feature Request Issue <https://github.com/openvinotoolkit/openvino/issues/new?assignees=octocat&labels=enhancement%2Cfeature&projects=&template=feature_request.yml&title=%5BFeature+Request%5D%3A+>`__
  if your idea is already well defined.

In both cases, provide a detailed description and list potential use cases,
benefits, and challenges. Keep in mind that even if your input is not immediately
prioritized, it may be used at a later or undertaken by the community.

> :fas:`code-branch` Contribute code changes

Reviewer: GFI could also be mentioned here

Author: GFI is emphasized in the tip admonition below, line 60.

Author: ![image](https://github.com/user-attachments/assets/e4df8aa5-f8ca-4f58-a070-a353cf368761)

and contacting `documentation maintainers <https://github.com/orgs/openvinotoolkit/teams/openvino-docs-maintainers>`__,
who will help you with information architecture and formatting, as well as
review, adjust, and merge the PR.

.. rubric:: Review user documentation
   :name: review-user-documentation

In most cases, creating a PR is enough to correct a documentation mistake, improve
the language, and update or extend the information. For your convenience, the
top-right panel of most pages includes the “Edit on GitHub” button that will
take you to the source file of the given article.

.. rubric:: Write new content
   :name: write-new-content

For more extensive changes in docs, reach out to any of the
`documentation maintainers <https://github.com/orgs/openvinotoolkit/teams/openvino-docs-maintainers>`__
to discuss the new content.

:fas:`bullhorn` Promote and support OpenVINO
############################################

.. rubric:: Popularize OpenVINO
   :name: popularize-openvino

Articles, tutorials, blog posts, demos, videos, and any other involvement in the
OpenVINO community is more than welcome. If you discuss or present OpenVINO on

> various social platforms, you are raising awareness of the product among A.I.

Reviewer: ```suggestion
various social platforms, you are raising awareness of the product among AI
```

enthusiasts and enabling other people to discover the toolkit.

Feel free to reach out to OpenVINO developers if you need help with making a
contribution. You can also contact
`documentation maintainers <https://github.com/orgs/openvinotoolkit/teams/openvino-docs-maintainers>`__
, if you need help with visuals, brand materials, or content creation in general.

.. rubric:: Help other community members
   :name: help-community

If you are an experienced OpenVINO user and want to help, you can share your
expertise with the community at any time. Check GitHub
`Discussions <https://github.com/openvinotoolkit/openvino/discussions>`__
and `Issues <https://github.com/openvinotoolkit/openvino/issues>`__ to see if
you can help someone.

.. note::

   By contributing to the OpenVINO project, you agree that your contributions
   will be licensed under `the terms of the OpenVINO repository <https://github.com/openvinotoolkit/openvino/blob/master/LICENSE>`__.

Additional Resources
#####################

> 

Reviewer: https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md#Technical-guide could also be added.

Author: > https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md#Technical-guide could also be added.

The article below, Code Contribution Guide is based on the technical guide you mention :)
